Assets/Script/Gameplay/Player/TrackPlayer.cs

```diff
@@ -417,6 +424,11 @@ protected override void UpdateVisuals(double visualTime)
                 CameraPositioner.Scoop();
             }
 
+            if (SettingsManager.Settings.EnableTrackEffects.Value && currentStarPowerAmount > _previousStarPowerAmount)
+            {
+                StarPowerEffect.PlayAnimation();
```

Comment:
Now that the effect is disabled by default, you'll need to enable it before calling its PlayAnimation.

```suggestion
                StarPowerEffect.gameObject.SetActive(true);
                StarPowerEffect.PlayAnimation();
```

(you might actually be able to have it activate itself in `PlayAnimation()` since you're calling a method directly rather than relying on Unity stuff, I'm not certain)